internal/cli/clients/client.go

```diff
@@ -1233,12 +1248,28 @@ func (c *EpinioClient) Push(ctx context.Context, name string, source string, par
 			return err
 		}
 
-		_, err = c.post(api.Routes.Path("ServiceBindingCreate", appRef.Org, appRef.Name), string(js))
+		b, err := c.post(api.Routes.Path("ServiceBindingCreate", appRef.Org, appRef.Name), string(js))
 		if err != nil {
 			return err
 		}
 
-		c.ui.Note().Msg("Done")
+		br := &models.BindResponse{}
+		if err := json.Unmarshal(b, br); err != nil {
+			return err
+		}
+
+		msg := c.ui.Note()
+		text := "Done"
+		if len(br.WasBound) > 0 {
+			text = text + ", With Already Bound Services"
```

Comment:
Should we write something like `With Already Bound Service[s]` or make it dependent on the number of services, because this can be confusing, if we mention serviceS and the user wonders if there are more then one.

Just a nitpick, I am also fine with this solution.

Reply:
Ok. I will fix the nit in some other PR, that way we do not have to through CI again.
Thanks for reviewing.